hack/tools.go

```diff
@@ -0,0 +1,16 @@
+// +build tools
+
+// This package contains code generation utilities
```

Comment:
This is a recommended way from `code-generator`, see https://github.com/kubernetes/code-generator/blob/master/tools.go